Index: src/main/java/Friendlist.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/Friendlist.java b/src/main/java/Friendlist.java
new file mode 100644
--- /dev/null	(date 1763314438589)
+++ b/src/main/java/Friendlist.java	(date 1763314438589)
@@ -0,0 +1,2 @@
+public class Friendlist {
+}
Index: src/main/java/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/Main.java b/src/main/java/Main.java
new file mode 100644
--- /dev/null	(date 1763312347371)
+++ b/src/main/java/Main.java	(date 1763312347371)
@@ -0,0 +1,35 @@
+import javax.swing.*;
+import java.awt.*;
+
+public class Main {
+    // TODO: Review where to place main
+    public static void main(String[] args) {
+        SwingUtilities.invokeLater(() -> new Main().createAndShowGUI());
+    }
+
+    private void createAndShowGUI(){
+
+        JFrame mainFrame = new JFrame("Dashboard");
+        mainFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+
+        // TODO: Fill w/ Info
+        JPanel dashboard = new JPanel();
+        mainFrame.add(dashboard, BorderLayout.CENTER);
+
+        // TODO: Fill w/ Info
+        JScrollPane leaderboard = new JScrollPane();
+        mainFrame.add(leaderboard, BorderLayout.EAST);
+
+        // TODO: Link to Other Windows
+        JPanel navBar = new JPanel(new FlowLayout(FlowLayout.CENTER));
+        String[] navLabels = {"Set Target", "Log Meal", "Save Recipes", "Profile"};
+        for (String label : navLabels) {
+            JButton button = new JButton(label);
+            navBar.add(button);
+        }
+        mainFrame.add(navBar, BorderLayout.SOUTH);
+
+        mainFrame.pack();
+        mainFrame.setVisible(true);
+    }
+}
Index: src/main/java/Dashboard.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/Dashboard.java b/src/main/java/Dashboard.java
new file mode 100644
--- /dev/null	(date 1763315189873)
+++ b/src/main/java/Dashboard.java	(date 1763315189873)
@@ -0,0 +1,36 @@
+import javax.swing.*;
+import java.awt.*;
+
+public class Dashboard extends JPanel {
+    private JLabel calories;
+    private JLabel remainingCalories;
+    private JLabel protein;
+    private JLabel carbs;
+    private JLabel fats;
+
+    Dashboard(){
+        //Todo: Replace w/ User obj
+        JPanel recipesPanel = new JPanel(new BorderLayout());
+        DefaultListModel<String> recipeModel = new DefaultListModel<>();
+        //Todo: Fill RecipeList
+        JList<String> recipeList = new JList<>(recipeModel);
+        recipesPanel.add(new JScrollPane(recipeList), BorderLayout.CENTER);
+
+        JPanel nutritionPanel = new JPanel();
+        nutritionPanel.setLayout(new BoxLayout(nutritionPanel, BoxLayout.Y_AXIS));
+
+        //TODO: Make Cleaner
+        nutritionPanel.add(calories);
+        nutritionPanel.add(remainingCalories);
+        nutritionPanel.add(protein);
+        nutritionPanel.add(carbs);
+        nutritionPanel.add(fats);
+
+        // View History
+        JButton viewHistoryButton = new JButton("View History");
+        nutritionPanel.add(viewHistoryButton);
+
+        JSplitPane splitPane = new JSplitPane(JSplitPane.HORIZONTAL_SPLIT, recipesPanel, nutritionPanel);
+        add(splitPane, BorderLayout.CENTER);
+    }
+}
